docs: update upstreaming state (#188)
nix-community/lanzaboote: README.md

## State of Upstreaming to Nixpkgs

> SecureBoot is available by adding this project in your configuration.

Reviewer: ```suggestion
SecureBoot is available by adding this project to your configuration.
```

> It works with NixOS 23.05 and NixOS unstable.
> 
> It relies on [bootspec](https://github.com/NixOS/rfcs/pull/125)
> which is enabled by default now.

Reviewer: ```suggestion
It relies on [bootspec](https://github.com/NixOS/rfcs/pull/125)
which is enabled by default since NixOS 23.05.
```.
